src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -507,4 +527,13 @@ private function isNewline(Token $token): bool
     {
         return $token->isWhitespace() && str_contains($token->getContent(), "\n");
     }
+
+    private function isMultilineConstructor(Tokens $tokens, int $index): bool
```

Comment:
The name is misleading - the constructor can be multiline with the method returning `false`.